python/semantic_kernel/processes/kernel_process/kernel_process_step_metadata.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+
+class KernelProcessStepMetadataAttribute:
+    """The metadata attribute for a Kernel Process Step."""
+
+    def __init__(self, version: str = "v1"):
+        """Initializes a new instance of the KernelProcessStepMetadataAttribute class."""
+        self.version = version
+
+
+def kernel_process_step_metadata(version: str = "v1"):
+    """Decorator to attach metadata to a Kernel Process Step class.
+
+    Example usage:
+        @kernel_process_step_metadata("v2")
```

Comment:
would be good to callout what a version refers to here, is the version related to the step, the process, the step state, the metadata definition, the metadata content?

Reply:
Good call. It's the version related to the step (or process as a step). Will improve this.